operator/apis/lifecycle/v1alpha1/common/common.go

```diff
@@ -31,6 +32,10 @@ const MaxWorkloadNameLength = 25
 const MaxTaskNameLength = 25
 const MaxVersionLength = 12
 
+var CannotCastKeptnAppVersionErr = errors.New("cannot cast KeptnAppVersion to v1alpha2")
```

Comment:
in the rest of the code base, we use `Err` as prefix. For example here: https://github.com/keptn/lifecycle-toolkit/blob/main/operator/controllers/errors/errors.go